### PR TITLE
Close modal when clicking outside

### PR DESCRIPTION
**Pull request by remy727**
### WHAT is this pull request doing?

Close modal when clicking outside

### CODE REVIEW

File: app/assets/javascripts/polaris_view_components.js [hunk 1]
```diff
@@ -833,7 +833,8 @@ class Modal extends Controller {
   }
   open() {
     this.element.classList.remove(this.hiddenClass);
-    this.element.insertAdjacentHTML("afterend", `<div class="${this.backdropClass}"></div>`);
+    const dropElement = `<div class="${this.backdropClass}" data-action="click->polaris#closeModal"></div>`;
```
Comment (remy727) [inline app/assets/javascripts/polaris_view_components.js:836]: @kirillplatonov, any idea why `closeModal` is not being triggered?

Comment (remy727) [reply app/assets/javascripts/polaris_view_components.js]: No worries. I fixed it.

**Comment by remy727** [comment]
@kirillplatonov, this PR is ready for review.

**Comment by remy727** [comment]
@kirillplatonov, can you please review this one when you have a chance?

**Comment by remy727** [comment]
Thank you 👍

**Comment by kirillplatonov** [comment]
Published v2.2.0 with latest updates: https://github.com/baoagency/polaris_view_components/releases/tag/v2.2.0

**Comment by remy727** [comment]
Great. Much appreciated!

**Pull request merged**